nixos/tests/sway.nix

```diff
@@ -66,30 +80,28 @@ import ./make-test-python.nix ({ pkgs, lib, ...} :
     machine.wait_for_file("/run/user/1000/wayland-1")
     machine.wait_for_file("/tmp/sway-ipc.sock")
 
-    # Test XWayland:
+    # Test XWayland (foot does not support X):
     machine.succeed(
         "su - alice -c 'swaymsg exec WINIT_UNIX_BACKEND=x11 WAYLAND_DISPLAY=invalid alacritty'"
     )
     machine.wait_for_text("alice@machine")
     machine.send_chars("test-x11\n")
     machine.wait_for_file("/tmp/test-x11-exit-ok")
-    print(machine.succeed("cat /tmp/test-x11.out"))
-    machine.screenshot("alacritty_glinfo")
+    machine.copy_from_vm("/tmp/test-x11.out")
+    machine.screenshot("alacrittyglinfo")
     machine.succeed("pkill alacritty")
 
-    # TODO: Switch to foot (or add an additional foot test) and use $mod+Return:
-    # Start a terminal (Alacritty) on workspace 3:
+    # Start a terminal (foot) on workspace 3:
     machine.send_key("alt-3")
-    machine.succeed(
-        "su - alice -c 'swaymsg exec WINIT_UNIX_BACKEND=wayland DISPLAY=invalid alacritty'"
-    )
+    machine.sleep(3)
```

Comment:
Out of curiosity: Is that sleep required when switching away from `swaymsg` or is this just out of precaution?

Reply:
I was running into issues of the terminal not opening when I didn't have it. Maybe it could be shorter

Reply:
Ok, thanks for all of your feedback. Should be fine, I was just curious.